src/url-handling.js

```diff
@@ -325,4 +319,19 @@ const _upgraders = {
       // Matches:  $1^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       .replace(/^(\/[^/]+\/[^/]+)\/markers\/?/, '$1/marker-table/');
   },
+  [3]: (processedLocation: ProcessedLocation) => {
+    const { query } = processedLocation;
+    // Removed "Platform only" checkbox from the stack chart.
+    if ('hidePlatformDetails' in query) {
+      delete query.hidePlatformDetails;
+      query.implementation = 'js';
+    }
+  },
 };
+
+if (Object.keys(_upgraders).length - 1 !== CURRENT_URL_VERSION) {
+  throw new Error(oneLine`
+    CURRENT_URL_VERSION does not match the numer of URL upgraders. If you added a
+    new upgrader, make sure and bump the CURRENT_URL_VERSION variable.
```

Comment:
nit: the expression `make sure and` sounds weird; maybe more idiomatic is `make sure you bumped...` ? (but of course you know better :) )